src/osd/osdepend.h

```diff
@@ -64,6 +64,7 @@ class osd_interface
 	// general overridables
 	virtual void init(running_machine &machine) = 0;
 	virtual void update(bool skip_redraw) = 0;
+	virtual void input_update(void) = 0;
```

Comment:
The `(void)` here is undesirable – C++ doesn’t support K&R functions, so you should use empty parentheses for a function that takes no arguments.

Reply:
Thanks.